blindml/backend/nni_helper.py

```diff
@@ -8,6 +8,7 @@
 from functools import cmp_to_key
 from pprint import pprint
 from subprocess import check_call, CalledProcessError, Popen, PIPE, STDOUT, call
+from sys import exit
```

Comment:
I don't think you need this? https://stackoverflow.com/questions/28621112/if-i-import-exit-from-sys-which-exit-will-it-run#comment45544678_28621112

Reply:
Hm. I added it for some reason, because I was having weirdness at that point. I'll investigate.